test/mastodon-toot-tests.el

```diff
@@ -1,6 +1,46 @@
 (require 'el-mock)
 
+(defconst mastodon-toot-multi-mention
+  '((mentions .
+              [((id . "1")
+                (username . "federated")
+                (url . "https://site.cafe/@federated")
+                (acct . "federated@federated.cafe"))
+               ((id . "1")
+                (username . "federated")
+                (url . "https://site.cafe/@federated")
+                (acct . "federated@federated.social"))
+               ((id . "1")
+                (username . "local")
+                (url . "")
+                (acct . "local"))])))
+
+(defconst mastodon-toot-no-mention
+  '((mentions . [])))
+
+(ert-deftest toot-multi-mentions ()
+  (should (string= (let ((mastodon-auth--acct-alist '(("https://local.social". "null")))
```

Comment:
I think this is more readable if you pull the `let` out.
That's basically the setup of the test context and now part of the actual test.